pytorch_lightning/plugins/plugins_registry.py

```diff
@@ -102,6 +102,9 @@ def get(self, name: str) -> Any:
             data = self[name]
             return data["plugin"](**data["init_params"])
 
+        if default is not None:
+            return default
+
```

Comment:
why do we need the default?

Reply:
That's the signature for the `get` method.
Or else, it raises a concern
```
pytorch_lightning/plugins/plugins_registry.py:93: error: Signature of "get" incompatible with supertype "Mapping"  [override]
```